cmd/queue/main.go

```diff
@@ -378,17 +378,10 @@ func supportsMetrics(ctx context.Context, logger *zap.SugaredLogger, env config)
 	return true
 }
 
-func buildAdminServer(logger *zap.SugaredLogger, healthState *health.State) *http.Server {
-	adminMux := http.NewServeMux()
-	drainHandler := healthState.DrainHandlerFunc()
-	adminMux.HandleFunc(queue.RequestQueueDrainPath, func(w http.ResponseWriter, r *http.Request) {
```

Comment:
This drops the pre-stop hook path - should we drop it from the revisions deployment spec?

https://github.com/knative/serving/blob/73af59be582a21827cd4d19d8c5bce99f3a3abe9/pkg/reconciler/revision/resources/deploy.go#L76-L88

Reply:
I don't think we should drop it at all :thinking:. This takes care of misbehaving user apps that don't correctly want to implement our "drain then wait X seconds" logic. Maybe we should add a test-image that doesn't handle SIGTERM at all to verify this behavior (that'd break now, I believe)